tests/ballerina-test-integration/src/test/java/org/ballerinalang/test/service/grpc/tool/ClientStubGeneratorTestCase.java

```diff
@@ -34,112 +36,123 @@
 /**
  * Protobuf to bal generation function testcase.
  */
-public class BalGenToolTest {
+public class ClientStubGeneratorTestCase {
+
     private static final String PACKAGE_NAME = ".";
     private static String protoExeName = "protoc-" + OSDetector.getDetectedClassifier() + ".exe";
     private static Path resourceDir = Paths.get(
-            BalGenToolTest.class.getProtectionDomain().getCodeSource().getLocation().getPath());
-    
+            ClientStubGeneratorTestCase.class.getProtectionDomain().getCodeSource().getLocation().getPath());
+
+    @BeforeClass
+    private void setup() {
+
```

Comment:
Extra new line?

Reply:
Fixed in f2bb0753488e0c357b61ea5a9d3639e43bdf9bfe